README.md

```diff
@@ -19,7 +19,7 @@ $ cargo binstall radio-sx128x@0.14.1-alpha.5
  WARN The package radio-sx128x v0.14.1-alpha.5 (x86_64-unknown-linux-gnu) has been downloaded from github.com
  INFO This will install the following binaries:
  INFO   - sx128x-util (sx128x-util-x86_64-unknown-linux-gnu -> /home/.cargo/bin/sx128x-util)
-Do you wish to continue? yes/[no]
+Do you wish to continue? yes/NO
```

Comment:
I prefer quoting yes to show that it's the default:

```suggestion
Do you wish to continue? [yes]/no
```

Reply:
OK, so to make sure I am getting you correctly, you would like YES to be the default, and you would like to mark it as [yes]/no.

I feel that is less clear, which is why I started my Issue.

But you must have thought deeply about this. So in the end, the markings shift, and [yes] will be implemented at the default. As you wish.

Reply:
yes, because this is how we showed `no` as the default option in the past, we want to make it consistent.

Reply:
Glad we got there in the end!